Cargo.toml

```diff
@@ -37,3 +37,7 @@ installers = []
 targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
 # Publish jobs to run in CI
 pr-run-mode = "plan"
+
+[patch.crates-io]
+# TODO remove me once https://github.com/LukeMathWalker/cargo-manifest/pull/57 is released
+cargo-manifest = { git = "https://github.com/hdoordt/cargo-manifest.git" }
```

Comment:
Remove this once <https://github.com/LukeMathWalker/cargo-manifest/pull/57> is released